.github/workflows/export_strings.yml

```
@@ -1,5 +1,12 @@
 name: Import strings
 on:
+  schedule:
+    - cron: '0 22 * * 3' # At 15:00 PDT on Wednesday
```

I'd move this earlier, not sure why FxA picked such a weird time.

9 PDT would work, as there's a chance I will see it before my day ends, and you'll have the entire day to look at it.
```suggestion
    - cron: '0 16 * * 3' # At 9:00 PDT on Wednesday
```